README.md

```diff
@@ -1,2 +1,29 @@
-# transporter
-Bulk file-transfer system for data ingest
+# Transporter
+Bulk file-transfer system for data ingest.
```

Comment:
I assume transporter will be used for data delivery as well and not just ingest?